docs: COPY does not support leading slash (#1457)
earthly/earthly: docs/earthfile/earthfile.md

The parameter `<src-artifact>` is an [artifact reference](../guides/target-ref.md#artifact-reference) and is generally of the form `<target-ref>/<artifact-path>`, where `<target-ref>` is the reference to the target which needs to be built in order to yield the artifact and `<artifact-path>` is the path within the artifact environment of the target, where the file or directory is located. The `<artifact-path>` may also be a wildcard.

The classical form of the `COPY` command differs from Dockerfiles in two cases:

* URL sources are not yet supported.

> * The current directory is not treated as the root directory; meaning sources in the current directory cannot be referenced with a leading `/`.

Reviewer: I feel that this is a bit clearer

```suggestion
* Absolute paths are not supported - sources in the current directory cannot be referenced with a leading `/`
```
